openreview/openreview.py

```diff
@@ -195,16 +197,19 @@ def get_tag(self, id):
         t = response.json()['tags'][0]
         return Tag.from_json(t)
 
-    def get_profile(self, email_or_id):
+    def get_profile(self, email_or_id = None):
         """
         Returns a single profile (a note) by id, if available
         """
-        tildematch = re.compile('~.+')
-        if tildematch.match(email_or_id):
-            att = 'id'
-        else:
-            att = 'email'
-        response = requests.get(self.profiles_url, params = {att: email_or_id}, headers = self.headers)
+        params = {}
+        if email_or_id:
+            tildematch = re.compile('~.+')
+            if tildematch.match(email_or_id):
+                att = 'id'
+            else:
+                att = 'email'
+            params[att] = email_or_id
+        response = requests.get(self.profiles_url, params = params, headers = self.headers)
```

Comment:
What happens if the params are empty? I don't understand the purpose of the change.

Reply:
the backend returns the profile of the logged user, the one that is represented in the token. 

I need to get the user profile when the client is created using the token and not the username/password.

```
client = openreview.Client(token = 'sdfsdfjskd589sdflkj')
```